book/guides/data_guide.md

```diff
@@ -0,0 +1,331 @@
+:::
+
+You can remove single files by using a gcsfs/fsspec filessytem as above
+
+```python
+import gcsfs
+
+fs = gcsfs.GCSFileSystem()  # equivalent to fsspec.fs('gs')
+fs.rm("leap-persistent/funky-user/file_to_delete.nc")
+```
+
+If you want to remove zarr stores (which are an 'exploded' data format, and thus represented by a folder structure) you have to recursively delete the store.
+
+```python
+fs.rm("leap-scratch/funky-user/processed_store.zarr", recursive=True)
+```
+
+## Transfering Data into Cloud Storage
+
+We distinguish between two primary *types* of data to upload: "Original" and "Published" data.
+
+- **Published Data** has been published and archived in a publically accessible location (e.g. a data repository like [zenodo](https://zenodo.org) or [figshare](https://figshare.com)). We do not recommend uploading this data to the cloud directly, but instead use [Pangeo Forge](https://pangeo-forge.readthedocs.io/en/latest/) to transform and upload it to the cloud. This ensures that the data is stored in an ARCO format and can be easily accessed by other LEAP members.
+- **Original Data** is any dataset that is produced by researchers at LEAP and has not been published yet. The main use case for this data is to share it with other LEAP members and collaborate on it. For original data we support direct uploaded to the cloud. *Be aware that original data could change rapidly as the data producer is iterating on their code*. We encourage all datasets to be archived and published before using them in scientific publications.
+
+(guides.data.ingestion)=
+
+### Ingesting Datasets into Cloud Storage
+
+If you do not find your dataset in the data catalog we can ingest it, so that the dataset is available for the LEAP community and beyond (see [](reference.infrastructure.buckets) for who can access which resource). Here at LEAP we want to ingest data in [Analysis-Ready Cloud-Optimized (ARCO)](reference.arco) formats.
+
+To start ingesting a dataset follow these steps:
+
+1. Add a new [dataset_request](https://github.com/leap-stc/data-management/issues/new?assignees=&labels=dataset&projects=&template=new_dataset.yaml&title=New+Dataset+%5BDataset+Name%5D) in the [data-management](https://github.com/leap-stc/data-management) repo so there is a central place where people can suggest datasets for ingestion and follow progress.
+1. Start a feedstock for your dataset. We organize any kind of data that is part of the [](explanation.architecture.data-library) in its own repository under the `leap-stc` github organization. Please use [our Template](https://github.com/leap-stc/LEAP_template_feedstock) to get started. Based on the 3 [types of data](explanation.data-policy.types) we host in the [](explanation.architecture.data-library) there are different ways of ingesting data (with specific instructions provided in the [template feedstock](https://github.com/leap-stc/LEAP_template_feedstock)):
```

Comment:
I think this is an information dense paragraph with almost every sentence linking to something. Could we break it up into maybe two bullet points with more explanation? 

Like: "start a feedstock for your dataset. A feedstock is ____". 

I think too many links also make things scarier? Creates impression that there are lots of moving parts; maybe we could explain/ link to "data library" at the top and then not have the text linked

Reply:
I would suggest sentence structure like: 

Any kind of data that is part of the [](explanation.architecture.data-library) is organized via its own repository under the `leap-stc` github organization. Pangeo Forge recipes are deployed via Github Actions in these repositories, referred to as [feedstocks](https://pangeo-forge.readthedocs.io/en/latest/deployment/feedstocks.html#what-is-a-feedstock). 

. Start a feedstock for your dataset. We organize any kind of data that is part of the [](explanation.architecture.data-library) . Please use [our Template](https://github.com/leap-stc/LEAP_template_feedstock) to get started. Based on the 3 [types of data](explanation.data-policy.types) we host in the [](explanation.architecture.data-library) there are different ways of ingesting data (with specific instructions provided in the [template feedstock](https://github.com/leap-stc/LEAP_template_feedstock)):